_operator_reference/security.md

```diff
@@ -115,7 +115,7 @@ oauth2:
 - `enabled`: true or false.
```

Comment:
These fields also support `encryptedFile...` secret references:
* `apiSecurity.ssl.keyStore`
* `apiSecurity.ssl.trustStore`
And `encrypted...` secret references:
* `apiSecurity.ssl.keyStorePassword`
* `apiSecurity.ssl.trustStorePassword`

Reply:
Ah. I didn't follow all the way to com.netflix.spinnaker.halyard.config.model.v1.security.SpringSsl